Not installable on Ubuntu 12.04

I wasn't abl to install ethercalc having Ubuntu 12.04 in place with system node and npm:

npm http 200 https://registry.npmjs.org/fresh/-/fresh-0.1.0.tgz
npm http 200 https://registry.npmjs.org/cookie/-/cookie-0.1.0.tgz
npm http 200 https://registry.npmjs.org/source-map
npm ERR! error installing uglify-js@2.3.6
npm ERR! error installing zappajs@0.4.22
npm ERR! error installing ethercalc@0.20121026.162049

npm ERR! Error: No compatible version found: source-map@'>=0.1.7- <0.2.0-'
npm ERR! Valid install targets:
npm ERR! ["0.0.0","0.1.0","0.1.1","0.1.2","0.1.3"]
npm ERR!     at installTargetsError (/usr/share/npm/lib/cache.js:488:10)
npm ERR!     at next_ (/usr/share/npm/lib/cache.js:438:17)
npm ERR!     at next (/usr/share/npm/lib/cache.js:415:44)
npm ERR!     at /usr/share/npm/lib/cache.js:408:5
npm ERR!     at saved (/usr/share/npm/lib/utils/npm-registry-client/get.js:147:7)
npm ERR!     at Object.oncomplete (/usr/lib/nodejs/graceful-fs.js:230:7)
npm ERR! You may report this log at:
npm ERR!     <http://bugs.debian.org/npm>
npm ERR! or use
npm ERR!     reportbug --attach /home/ethercalc/npm-debug.log npm
npm ERR! 
npm ERR! System Linux 3.5.0-40-generic
npm ERR! command "node" "/usr/bin/npm" "install" "ethercalc"
npm ERR! cwd /home/ethercalc
npm ERR! node -v v0.6.12
npm ERR! npm -v 1.1.4
npm ERR! message No compatible version found: source-map@'>=0.1.7- <0.2.0-'
npm ERR! message Valid install targets:
npm ERR! message ["0.0.0","0.1.0","0.1.1","0.1.2","0.1.3"]
npm http 200 https://registry.npmjs.org/buffer-crc32/-/buffer-crc32-0.2.1.tgz
npm http 200 https://registry.npmjs.org/cookie-signature/-/cookie-signature-1.0.1.tgz
npm http 200 https://registry.npmjs.org/send/-/send-0.1.0.tgz
npm http 200 https://registry.npmjs.org/policyfile/-/policyfile-0.0.4.tgz
npm http 200 https://registry.npmjs.org/debug/-/debug-0.7.2.tgz
npm http 200 https://registry.npmjs.org/base64id/-/base64id-0.1.0.tgz
npm http 200 https://registry.npmjs.org/socket.io-client/-/socket.io-client-0.9.16.tgz
npm http 200 https://registry.npmjs.org/optimist/-/optimist-0.3.7.tgz
npm http 200 https://registry.npmjs.org/redis/-/redis-0.7.3.tgz
npm http 200 https://registry.npmjs.org/optparse/-/optparse-1.0.3.tgz
npm http 200 https://registry.npmjs.org/stylus/-/stylus-0.27.2.tgz
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/ethercalc/npm-debug.log
npm not ok

--- a/README.mkdn
+++ b/README.mkdn
@@ -4,7 +4,7 @@
     * English: https://gist.github.com/3978463
     * 中文版: https://gist.github.com/3985324
 * Language: [LiveScript](http://livescript.net/)
-* Runtime: [Node.js](http://nodejs.org/) (0.8+ preferred; compatible with 0.4)
+* Runtime: [Node.js](http://nodejs.org/) (0.8+ preferred)
 * Services: [Redis](http://redis.io) (2.4+; fall-back to on-disk JSON storage if not present)
     * Multi-server is supported _only_ when running with Redis
 * Browsers tested: Safari, Chrome, Firefox, IE.

